src/Resources/config/routing/tree.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <routes xmlns="http://symfony.com/schema/routing" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/routing http://symfony.com/schema/routing/routing-1.0.xsd">
     <route id="sonata_admin_tree" path="/tree">
-        <default key="_controller">sonata.admin.doctrine_phpcr.tree_controller:treeAction</default>
+        <default key="_controller">sonata.admin.doctrine_phpcr.tree_controller::treeAction</default>
```

Comment:
i think this is another reason to raise the min symfony version to 3.4. otherwise we need a parameter for this or 2 different files to also have the single colon syntax. and thats not worth it imho.

Reply:
does this really work with symfony 2?